korman/exporter/animation.py

```diff
@@ -72,7 +72,11 @@ def fetch_animation_data(id_data):
         # There is a race condition in the client with animation loading. It expects for modifiers
         # to be listed on the SceneObject in a specific order. D'OH! So, always use these funcs.
         agmod, agmaster = self.get_anigraph_objects(bo, so)
-        atcanim = self._mgr.find_create_object(plATCAnim, so=so)
+        anim_data = bo.plasma_modifiers.animation
```

Comment:
`anim_data` is playing with my mind a little bit. I think because there is a blender type named `AnimationData`. Could we change this to something like `anim_mod`?

Reply:
Sure!